tests/integration/census/test_census_household_submission_data.py

```diff
@@ -17,14 +17,6 @@ def test_census_household_data_matches_census_individual(self):
     def get_expected_submission_data():
         expected_downstream_data = [
             {
-                'answer_id': 'building',
```

Comment:
You also need to remove the answer from https://github.com/ONSdigital/eq-survey-runner/blob/9d9ac4a41b98eaa126c4a851867f9f3abcd4af07/tests/integration/census/test_census_household_submission_data.py#L1181

Reply:
done :)